packages/connect-web-test/src/gen/connectrpc/conformance/v1/messages_pb.ts

```diff
@@ -30,7 +30,7 @@
 
 // Message definitions to be used by integration test service definitions.
 
-// @generated by protoc-gen-es v1.5.0 with parameter "ts_nocheck=false,target=ts"
+// @generated by protoc-gen-es v1.4.2 with parameter "ts_nocheck=false,target=ts"
```

Comment:
You need to merge in main again or rebase, re-generate code an re-format with the updated prettier, otherwise, this will likely fail CI when merged to main. 

I guess this will also make the diff here much smaller, and make it easier to follow the actual change.

Reply:
It was not the merge, took me some time to figure out. The npm version resolution logic kept installing `1.4.2` on this branch but is `1.5.0` in main. It also added the integrity values this time around. Should be good now